packages/blockchain-wallet-v4-frontend/src/scenes/Login/components/UrlNoticeBar/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Icon, Link, Text } from 'blockchain-info-components'
+import { isMobile, media } from 'services/styles'
+
+const BACKGROUND_COLOR = '#0C6CF2'
+const FONT_COLOR = 'white'
+const SIZE = isMobile() ? '12px' : '14px'
```

Comment:
This is kinda weird. Could you add this directly to the component? Or maybe add this check to the component itself? I don't think that this `isMobile` quite work like that, outside the component

Reply:
I'm adding it here to avoid code repetition, if it's ok with you then I'll change it, sure.

Reply:
thanks!!

Reply:
well, i don't know if the `isMobile` is going to get updated if the user changes the window, or if it's set to stone when it's defined.

Reply:
and btw, use `rem`

Reply:
Done